README.md

```diff
@@ -48,6 +48,17 @@ pip install astro-sdk-python[amazon,google,snowflake,postgres]
     airflow db init
     ```
 
+    > **Note:** `AIRFLOW__CORE__ENABLE_XCOM_PICKLING` needs to be enabled for `astro-sdk-python`.
+
+ Currently, custom XCom backends are limited to data types that are json serializable. Since Dataframes are not json serializable, we need to enable XCom pickling to store dataframes.
+
+The data format used by pickle is Python-specific. This has the advantage that there are no restrictions imposed by external standards such as JSON or XDR (which can’t represent pointer sharing); however it means that non-Python programs may not be able to reconstruct pickled Python objects.
```

Comment:
Do we need this?

Reply:
It would add in context to user for why we need this.

Reply:
>however it means that non-Python programs may not be able to reconstruct pickled Python objects.

I mean this line, we don't need that one ^^. But I will let you decide

Reply:
Maybe if someone wants to opt-out and use airflow rest api?

Reply:
I did not understand @feluelle. Could you elaborate on this, please?

Reply:
If you read data from xcom via the airflow rest api you would not be able to reconstruct it or? Maybe only with a python rest api client - but not the go client for example or? (that is what the sentence is saying, right? So maybe it is useful to leave it there.)